ui/chatview.go

```diff
@@ -386,6 +387,10 @@ func (chatView *ChatView) formatMessageAuthor(message *discordgo.Message) string
 
 func (chatView *ChatView) formatMessageText(message *discordgo.Message) string {
 	if message.Type == discordgo.MessageTypeDefault {
+		var partial = badEmoji.FindAllStringSubmatch(message.Content, -1)
+		for i := len(partial)-1; i>=0; i--	{
+			message.Content = strings.Replace(message.Content, partial[i][0], partial[i][1], -1)
+		}
```

Comment:
* Use a range-for-loop
* Put the code into `chatView.formatDefaultMessageText`
* Refrain from using `-1` in the `Replace` call.
* Make sure that you do the escaping in order to avoid breaking emojis in code blocks or something like that